src/components/vm/consoles/vncBody.jsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright 2024 Fsas Technologies Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+    FormGroup, FormHelperText, HelperText, HelperTextItem,
+    InputGroup, TextInput, Button, Checkbox
+} from "@patternfly/react-core";
+import { EyeIcon, EyeSlashIcon } from "@patternfly/react-icons";
+
+import cockpit from 'cockpit';
+
+const _ = cockpit.gettext;
+
+export const VncRow = ({ idPrefix, onValueChanged, dialogValues, validationErrors }) => {
+    const [showPassword, setShowPassword] = useState(false);
+
+    return (
+        <>
+            <FormGroup
```

Comment:
With the dialog showing only the port, this could use an info alert/note that the server will only listen to localhost, and that you need to use ssh tunneling for remote access. If there's documentation about that somewhere, we should link to that. If not, let's not block on that, and people who don't know about that will then simply not use that feature -- they are no worse off than before.

Reply:
> With the dialog showing only the port, this could use an info alert/note that the server will only listen to localhost, and that you need to use ssh tunneling for remote access.

Yes, later. We need to dig into qemu.conf to figure out the details.